client/src/components/ProductsList.js

```diff
@@ -46,16 +52,25 @@ export default class ProductsList extends Component {
     });
     console.log(this.state.searchCategory);
     console.log(this.state.searchText);
+    console.log(this.state.priceValue);
   };
 
-  render() {
-    const useStyles = makeStyles(theme => ({
-      card: {
-        maxWidth: 100
-      }
-    }));
+  handlePriceChange = (event, newValue) => {
+    this.setState({
+      priceValue: newValue
+    });
+  };
 
-    const classes = useStyles;
+  handleDateChange = date => {
+    this.setState({
+      selectedDate: date
+    });
+  };
+
+  render() {
```

Comment:
@ninetteadhikari I am getting the fake data in the backend via postman but I can't get your page rendered. Am I missing something?

```json
 {
        "location": {
            "latitude": -18.0912,
            "longitude": 70.6121
        },
        "imageUrl": [
            "http://lorempixel.com/640/480/business",
            "http://lorempixel.com/640/480/business",
            "http://lorempixel.com/640/480/business",
            "http://lorempixel.com/640/480/business",
            "http://lorempixel.com/640/480/business"
        ],
        "tags": [
            "tablets",
            "chair",
            "kitchen",
            "cabinet",
            "reception"
        ],
        "wishlist": [
            "5d97c695eabd3708476fc8f7",
            "5d97c695eabd3708476fc8f2",
            "5d97c695eabd3708476fc8f0"
        ],
        "requested": [
            "5d97c695eabd3708476fc8f2",
            "5d97c695eabd3708476fc8f2"
        ],
        "_id": "5d97c8c21abc6708eb9eca53",
        "title": "Aut fuga sint nemo.",
        "description": "Ex amet nesciunt dolore eum sint. Sequi et non unde explicabo provident veniam voluptas unde beatae. Molestiae nulla est eos nobis.",
        "brand": "Madame Stoltz",
        "category": "Carpentry",
        "quantity": 8,
        "price": 131.22,
        "currency": "JMD",
        "company": "Thompson - Feeney",
        "availability": "2020-02-14T07:25:00.168Z",
        "warrantyUntil": "2020-08-04T06:02:53.707Z",
        "condition": "used",
        "isSold": false,
        "seller": "5d97c695eabd3708476fc8f3",
        "buyer": "5d97c695eabd3708476fc8f7",
        "__v": 0
    }
```

Reply:
Fixed. :D